packages/hoppscotch-backend/src/infra-config/infra-config.service.ts

```diff
@@ -118,9 +152,14 @@ export class InfraConfigService implements OnModuleInit {
     if (E.isLeft(isValidate)) return E.left(isValidate.left);
 
     try {
+      const { isEncrypted } = await this.prisma.infraConfig.findUnique({
+        where: { name },
+        select: { isEncrypted: true },
+      });
```

Comment:
Does this DB call need to be made? can't we just get this info from `getDefaultInfraConfigs`.

Reply:
`getDefaultInfraConfigs` also has a DB (`count`) call. So, ultimately it doesn't help much.